File: 110algoritmos/exercicios014/ex014-Csharp/ex014.cs
using System;

class Program{

    static void Main() {

        double c, f;

        Console.Write("Informe a temperatura em Celsius: ");
        c = double.Parse(Console.ReadLine());

        f = (9 * c)/5 + 32;

        Console.WriteLine("A temperatura em " + c + " C corresponde em " +f+ " F");

    }
}

File: 110algoritmos/exercicios014/ex014-Rust/ex014.rs
use std::io;

fn main() {
    let mut c:String = String::from("");
    let f:f32;

    println!("Informe a temperatura em Celsius");
    io::stdin()
        .read_line(&mut c)
        .expect("Erro na leitura de C");
    let c:f32 = c.trim().parse().expect("Erro na conversao de C");

    f = (9.0 * c ) / 5.0 + 32.0;

    println!("A temperatura em {} C corresponde {} F", c, f);
}